### PR TITLE
A propsal for enums

### PR DESCRIPTION
**Pull request by ahmadmayahi**

### CODE REVIEW

File: accepted/PSR-1-basic-coding-standard.md [hunk 1]
```diff
@@ -97,14 +97,14 @@ if (! function_exists('bar')) {
 }
 ~~~
 
-## 3. Namespace and Class Names
+## 3. Namespace, Class Names and Enums
```
Comment (drealecs) [inline accepted/PSR-1-basic-coding-standard.md:100]: I think the only needed change is in section 4.:
Changing
```
The term "class" refers to all classes, interfaces, and traits.
```
to
```
The term "class" refers to all classes, interfaces, enums and traits.
```
Or something similar.

Comment (drealecs) [reply accepted/PSR-1-basic-coding-standard.md:100]: But we can also consider that enum is just a special type of class. So no changes would be needed.

**Comment by Crell** [comment]
We don't modify already-approved specs.  But as @drealecs noted, class includes enums for most practical purposes.

(Except for the brace-on-own-line bit, because that's intensely stupid for classes already and makes even less sense on enums, but that's a different matter.)

**Comment by Jean85** [comment]
Closing this as already explained by Crell.

**Pull request closed without merging**